.github/workflows/e2e-fulltests-ci.yml

```diff
@@ -0,0 +1,83 @@
+---
+  name: E2E Tests
+  on:
+    # For PRs, this workflow gets triggered from the Argo Events platform.
+    # Check the following repo for details: https://github.com/mattermost/delivery-platform
```

Comment:
@mvitale1989 Which is the workflow of delivery-platform that triggers this workflow? Looking to see what are the inputs.

Reply:
I haven't implemented it yet, I can add you as a reviewer to the corresponding PR as soon as I push it if you'd like.

FYI I mentioned the plan in [this comment from the corresponding ticket](https://mattermost.atlassian.net/browse/CLD-7198?focusedCommentId=161767), it should roughly follow this logic:
1. Create an argo-events Sensor that listen for [`commit_comment` events](https://docs.github.com/en/webhooks/webhook-events-and-payloads#commit_comment), to handle comments from PRs
2. Ensure the comment author belongs to the `mattermost` org (which is the same as mattermod does currently)
3. If the comment is e.g. `/e2e-test`, then this workflow is triggered

Reply:
Btw I added these two comment lines because we plan to implement this Sensor soon for the developers' ease-of-use, but the delivery-platform part of this is actually unrelated to this PR. It just builds on top of the mechanism implemented here.

We can safely merge this to already offer the functionality through the Github WebUI, and then implement the trigger-by-comment later ([related ticket](https://mattermost.atlassian.net/browse/CLD-7198))

Reply:
@esarafianou i'll resolve this conversation, but feel free to reopen it if you have more questions on this